src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -796,76 +801,79 @@ private static OpenApiMediaType CreateResponseContainer(string responseObjectSch
         {
             Dictionary<string, OpenApiSchema> schemas = new();
 
-            foreach (KeyValuePair<string, DatabaseObject> entityDbMetadataMap in _metadataProvider.EntityToDatabaseObject)
+            foreach (ISqlMetadataProvider metadataProvider in _metadataProviderFactory.ListMetadataProviders())
```

Comment:
Same no CosmosSqlMetadataProvider should undergo this for loop

Reply:
discussed offline. going to remove loop as this pr is not addressing rest support for multi-db. using default metadataprovider.